manifests/cf-manifest/spec/manifest/instance_group_spec.rb

```diff
@@ -61,20 +61,6 @@ def get_instance_group(instance_group_name)
     end
   end
 
-  describe "in order to match the upstream Diego instance_group ordering" do
-    it "has diego-api before scheduler" do
-      expect("diego-api").to be_ordered_before("scheduler")
-    end
-
-    it "has scheduler before the diego cells" do
-      expect("scheduler").to be_ordered_before("diego-cell")
-    end
-
-    it "has api before scheduler" do
-      expect("api").to be_ordered_before("scheduler")
-    end
-  end
```

Comment:
> We'll soon be using upstream manifests as a base, so the small aesthetic
> difference that ordering makes will be moot.

The reason for asserting on the ordering is not aesthetic. We had an incident a while back because we had the order different to upstream, and upgraded the cells before bbs. This test was added in de70ecd to help catch this in future.

I'm inclined to leave this test in place until we're actually consuming the upstream manifests directly.

Reply:
:+1: OK I can re-order and re-instate the tests